inc/entity.class.php

```diff
@@ -2981,10 +2982,12 @@ static function generateLinkSatisfaction($ticket) {
    **/
    static function getAutoAssignMode($val = null) {
 
-      $tab = [self::CONFIG_PARENT                  => __('Inheritance of the parent entity'),
-                   self::CONFIG_NEVER                   => __('No'),
-                   self::AUTO_ASSIGN_HARDWARE_CATEGORY  => __('Based on the item then the category'),
-                   self::AUTO_ASSIGN_CATEGORY_HARDWARE  => __('Based on the category then the item')];
+      $tab = [
+         self::CONFIG_PARENT                  => __('Inheritance of the parent entity'),
+         self::CONFIG_NEVER                   => __('No'),
+         self::AUTO_ASSIGN_HARDWARE_CATEGORY  => __('Based on the item then the category'),
+         self::AUTO_ASSIGN_CATEGORY_HARDWARE  => __('Based on the category then the item'),
+      ];
```

Comment:
I wanted to create a distinct auto assign  setting for each itemtype (ticket, change and problem), but it was decided to mutualize the existing one for all. I kept the code style.